src/coreclr/scripts/superpmi.py

```diff
@@ -1636,20 +1636,22 @@ def determine_pmi_location(coreclr_args):
 
 def determine_jit_name(coreclr_args):
     """ Determine the jit based on the OS. If "-altjit" is specified, then use the specified altjit,
-        or an appropriate altjit based on target.
+        or an appropriate altjit based on target. This function is called for cases where the
+        "-altjit" flag is not used, so be careful not to depend on the "altjit" attribute existing.
 
     Args:
         coreclr_args (CoreclrArguments): parsed args
 
     Return:
-        jit_name(str) : name of the jit for this os
+        jit_name(str) : name of the jit for this OS
     """
 
     jit_base_name = "clrjit"
-    if isinstance(coreclr_args.altjit, str):
-        jit_base_name = coreclr_args.altjit
-    elif coreclr_args.altjit == True:
-        jit_base_name = "protononjit"
+    if hasattr(coreclr_args, "altjit"):
+        if isinstance(coreclr_args.altjit, str):
+            jit_base_name = coreclr_args.altjit
+        elif coreclr_args.altjit == True:
+            jit_base_name = "protononjit"
```

Comment:
won't this be following the new nomenclature now like `clrjit_*`?

Reply:
yes. the default needs to get more generally updated for the new cross names, but you can specify the names manually now.

Reply:
I didn't realize until now that I have https://github.com/dotnet/runtime/pull/42133 still open, but in that PR, I was trying to remove the references of `protononjit`. Can you do something similar with regards to default name?

Reply:
ok, I changed it to require that `-altjit` take a file name. I don't think it makes sense to choose a default because there are too many options and it's not clear what the best option would be as a default.